rest/server_context.go

```diff
@@ -82,6 +83,27 @@ func NewServerContext(config *ServerConfig) *ServerContext {
 		sc.startStatsReporter()
 	}
 
+	if config.Replications != nil {
+		go func() {
+			time.Sleep(kServerStartupWait)
```

Comment:
The local REST endpoints do not become available immediately, if replications reference local DB's then they may (most of the time they do) fail to connect to the DB's on startup.

Continuous replications will recover, but one shot replications will simply fail.